### ISSUE TITLE
Undo button still clickable during undo animation

### ISSUE THREAD

**Issue by SimplicityApks** [nhaarman/ListViewAnimations#343]
### Steps to reproduce
* ListView with a SimpleSwipeUndoAdapter (occurs in the sample app too if you're fast)
* Swipe away an item to reveal the undo bar
* Hit the undo button twice in rapid succession

### Result
* The onUndo() callback in the adapter is called twice
* The undo swipe in animation is restarted (list item gets set back off screen and swipes in again)

### Expected Result
* onUndo() callback called only once, animation not resetted
* the undo button should not be clickable during the animation

**Comment by SimplicityApks** [comment]
Also, it would be nice to have touch feedback on the undo button (apart from me wanting to have the layout like in the GMail app per default...), but that is probably a larger issue for another time, as it requires a totally different implementation...
Still great library, keep up the good work!
